R/set_tag.R

```diff
@@ -1,14 +1,16 @@
-#' Set a New Tag and Create a Release
+#' Set a New Tag
 #'
 #' This function only works when run in a GitHub Action on the master branch.
 #' Otherwise it will only return a message.
-#' @export
+#' It sets new tag a the current commit using the related entry from `NEWS.md`
+#' as message.
+#' This tag will turn into a release.
```

Comment:
Is this a helper function that is mainly used internally?  If not, it could be interesting to add a link where to find more information on tags and releases.  (Keep in mind that this package may be used by INBO users that never setup packages or repositories before.)